Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -347,3 +337,15 @@ extension AutocompleteTextField: MenuHelperInterface {
         autocompleteDelegate?.autocompletePasteAndGo(self)
     }
 }
+
+extension AutocompleteTextField: ThemeApplicable {
+    func applyTheme(theme: Theme) {
+        backgroundColor = theme.colors.layer3
+        textColor = theme.colors.textPrimary
+        tintColor = theme.colors.layerAccentNonOpaque
+        if self.autocompleteTextLabel?.attributedText != nil {
+            self.autocompleteTextLabel?.backgroundColor = backgroundColor
+            self.autocompleteTextLabel?.textColor = textColor
```

Comment:
```suggestion
        if autocompleteTextLabel?.attributedText != nil {
            autocompleteTextLabel?.backgroundColor = backgroundColor
           autocompleteTextLabel?.textColor = textColor
```

Reply:
I was working on a mac mini for a while and having self. was the only way to assess if a variable is local/global at a glance, until Xcode would load the colors and show everything. Could still happen on slower/older computers or projects that take forever to build.